addons/website_forum/controllers/website_forum.py

```diff
@@ -382,6 +382,8 @@ def post_create(self, forum, post_parent=None, **post):
             'parent_id': post_parent and post_parent.id or False,
             'tag_ids': post_tag_ids
         })
+        if post_parent:
+            post_parent.bump(force=True)
```

Comment:
post_parent.sudo().bump maybe ? or improve bump method with a less complicated condition, but we should never give an argument allowing to write as sudo on public methods.